modules/ROOT/pages/provisioning-aws.adoc

```diff
@@ -102,7 +102,36 @@ ignition:
       source: s3://instance1-infra/bootstrap.ign
 ----
 
-Then you can launch the instance using the same command as xref:#_customized_example[], passing the minimal configuration you just created.
+Then you can launch the instance using the following command, passing the minimal configuration you just created.
+
+Note: You need to create an IAM profile with `s3:GetObject` permission, and attach the role to the instance.
+
+.Launching and customizing a new instance with remote Ignition file from a S3 bucket
+[source,bash]
+----
+NAME='instance1'
+SSHKEY='my-key'         # the name of your SSH key: `aws ec2 describe-key-pairs`
+IMAGE='ami-xxx'         # the AMI ID found on the download page
+DISK='20'               # the size of the hard disk
+REGION='us-east-1'      # the target region
+TYPE='m5.large'         # the instance type
+SUBNET='subnet-xxx'     # the subnet: `aws ec2 describe-subnets`
+SECURITY_GROUPS='sg-xxx' # the security group `aws ec2 describe-security-groups`
+USERDATA='/path/to/config.ign' # path to your Ignition config
+PROFILE='xxx-profile'    # the name of an IAM instance profile
+aws ec2 run-instances                     \
+    --region $REGION                      \
+    --image-id $IMAGE                     \
+    --instance-type $TYPE                 \
+    --key-name $SSHKEY                    \
+    --subnet-id $SUBNET                   \
+    --security-group-ids $SECURITY_GROUPS \
+    --user-data "file://${USERDATA}"      \
+    --associate-public-ip-address         \
```

Comment:
We should document why this is needed

Reply:
Maybe we do not need `--associate-public-ip-address` if you are using a nondefault VPC, see https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-instance-addressing.html#concepts-public-addresses

Reply:
Remote it as it is not necessary if you use private subnet with NAT.